hasher-matcher-actioner/hmalib/writebacker/writebacker_base.py

```diff
@@ -157,7 +157,7 @@ def te_api(self) -> ThreatExchangeAPI:
         mock_te_api = os.environ.get("MOCK_TE_API")
         if mock_te_api == "True":
             return MockedThreatExchangeAPI()
-        api_token = AWSSecrets().te_api_token()
+        api_token = AWSSecrets("").te_api_token()
```

Comment:
I don't see the existing writebacker infra working with the signal exchange API abstraction. We might have to rewrite this completely. 

So, I didn't invest time in bringing the prefix all the way to this module.

Reply:
This will still work because as clarified in the aws_secrets.py docstring, the prefix is not used for te_api_token() or hma_api_tokens()